mycli/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "mycli" %}
+{% set version = "1.19.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 354119ab5b0462bb6a0e9f572515b573e2fc6155934beb06f038c7bd09fdbb74
+
+build:
+  number: 0
+  entry_points:
+    - mycli = mycli.main:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - backports.csv
```

Comment:
I think most of these dependencies aren't needed in the `host` section. You probably only need:
```yaml
requirements:
  host:
    - python
    - pip
```

Reply:
Fixed!